src/modals/_SearchModal.jsx

```diff
@@ -112,26 +112,20 @@ const Search = ({ fetchSearchItems, onItemSelected }) => {
   }, []);
 
   return (
-    <ContentContainer
+    <ApplicationModal
       className={cx(theme.className)}
```

Comment:
Is the theme context handed by ApplicationModal? Do we need to grab it here and pass it down?

Reply:
The modal doesn't directly need the theme context it just happens to be the first div wrapping all search content. We could have the same effect by adding it to the toolbar and search items independently.

Reply:
ApplicationModal does not take custom props, so I don't think this is doing anything right now.

Reply:
lol